app/javascript/components/public_health/AdvancedFilter.js

```diff
@@ -30,7 +30,6 @@ class AdvancedFilter extends React.Component {
           description: 'Recipients who had a report created in the last 24 hours',
           type: 'boolean',
         },
-        { name: 'paused', title: 'Notifications Paused (Boolean)', description: 'Recipients who have paused notifications', type: 'boolean' },
```

Comment:
@kevstoy @ChristineDuong 
This could make users completely blind in the system of which patients have paused their notifications. Did they specifically ask for this like removing the pause button from the patient show page?

Reply:
We ended up removing (commenting out) the "Pause Notifications" option for patients, so I thought this filter would be confusing. 
![Screen Shot 2020-12-09 at 9 15 43 AM](https://user-images.githubusercontent.com/6525651/101640932-1f546380-39ff-11eb-8cc1-efea44624f99.png)

But I understand the point either way. @kevstoy